React/Views/RCTView.m

```diff
@@ -1169,6 +1172,17 @@ - (void)displayLayer:(CALayer *)layer
   backgroundColor = _backgroundColor.CGColor;
 #endif
 
+  #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+  CATransform3D transform = self.transform3D;
```

Comment:
i am bit of a dot notation hater

Reply:
Also this is the old ISS tag. I'll update.